### PR TITLE
Added Calendar App

### PR DESCRIPTION
**Pull request by shalini-bhandari**
Closes: #830 

- **Title:**  Calendar App
- **Name:** Shalini Bhandari
- **Idenitfy yourself:**  gssoc-ext, hacktoberfest

### Screenshots 📷

Screenshot 2024-10-11 at 11 09 17

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by shalini-bhandari** [comment]
@Avdhesh-Varshney kindly review it and let me know if any changes are to be made

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Keep your project in `Vanilla-JS-Projects/Intermediate`.

**Comment by shalini-bhandari** [comment]
@Avdhesh-Varshney kindly review it again. 
Regards

File: Front-end-Projects/README.md
Comment (Avdhesh-Varshney) [inline Front-end-Projects/README.md:1]: Undo this project listing from here.
Enlist your project in Vanilla js readme file

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**